### PR TITLE
Build a presentation for stakeholders onboarding

### PR DESCRIPTION
**Pull request by frahugo**
image

### CODE REVIEW

**Comment by frahugo** [comment]
@nicholasjhenry This is the result of our last workshop.  I think this is quite enough for now, we can improve it on the next project.

File: process/02-onboarding/stakeholder-presentation-en.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+# Agenda
+
+- The nature of software development
+- Risks in developing software
+- Practices we embrace
+
+---
+
+> Software creation is an empirical activity
```
Comment (nicholasjhenry) [inline process/02-onboarding/stakeholder-presentation-en.md]: Do we need to example on that more? Here are some notes I have:

# Why the pain?

* Waterfall is the wrong process
* Suited for manufacturing (planned process)
* Not suited for new product development (empirical process)

^ The empirical model of process control provides and exercises control through frequent inspection and adaptation for processes that are imperfectly defined and generate unpredictable and unrepeatable outputs. 

# Empirical Process #

## "imperfectly defined and generate unpredictable and unrepeatable outputs"

# Empirical Process (new product development) #

File: process/02-onboarding/stakeholder-presentation-en.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+![inline](./triangle.png)
+
+---
+
+# Risks in software development
+
+- Going over budget
+- Late delivery
+- Not meeting requirements/expectations
+  - features, performance, reliability, durability
```
Comment (nicholasjhenry) [inline process/02-onboarding/stakeholder-presentation-en.md]: Seems like these should be bullet points, but we are using commas?

File: process/02-onboarding/stakeholder-presentation-en.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+![inline](./triangle.png)
+
+---
+
+# Risks in software development
+
+- Going over budget
+- Late delivery
+- Not meeting requirements/expectations
```
Comment (nicholasjhenry) [inline process/02-onboarding/stakeholder-presentation-en.md]: We should number these (1. 2. 3.) -- as in the BIG 3.

File: process/02-onboarding/stakeholder-presentation-en.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+---
+
+# Practices we embrace
+
+- Dedicated Product Owner
+- Iterative approach
+- Feedback early and often
+- Executable specifications
+- Open source
```
Comment (nicholasjhenry) [inline process/02-onboarding/stakeholder-presentation-en.md]: In the notes, we should have the benefits of these and how they help reduce the risk of the big 3 problems.

**Comment by nicholasjhenry** [comment]
@frahugo can we add the photos of the notes to this issue, please?

**Comment by frahugo** [comment]
@nicholasjhenry you can have another look if you want.

**Pull request merged**